seacrowd/sea_datasets/wisesight_thai_sentiment/wisesight_thai_sentiment.py

```diff
@@ -0,0 +1,184 @@
+* More characteristics of the data can be explore: https://github.com/PyThaiNLP/wisesight-sentiment/blob/master/exploration.ipynb
+"""
+
+_SOURCE_VIEW_NAME = DEFAULT_SOURCE_VIEW_NAME
+_UNIFIED_VIEW_NAME = DEFAULT_SEACROWD_VIEW_NAME
+
+_HOMEPAGE = "https://github.com/PyThaiNLP/wisesight-sentiment"
+
+_LANGUAGES = ["tha"]  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+
+
+_LICENSE = "Creative Commons Zero v1.0 Universal license"
+
+
+_LOCAL = False
+
+
+_URLS = {
+    _DATASETNAME: "https://github.com/PyThaiNLP/wisesight-sentiment/raw/master/huggingface/data.zip",
+}
+
+# TODO: add supported task by dataset. One dataset may support multiple tasks
+_SUPPORTED_TASKS = [Tasks.SENTIMENT_ANALYSIS]  # example: [Tasks.TRANSLATION, Tasks.NAMED_ENTITY_RECOGNITION, Tasks.RELATION_EXTRACTION]
+
+# TODO: set this to a version that is associated with the dataset. if none exists use "1.0.0"
+#  This version doesn't have to be consistent with semantic versioning. Anything that is
+#  provided by the original dataset as a version goes.
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class WisesightSentiment(datasets.GeneratorBasedBuilder):
+    """Wisesight Sentiment Corpus: Social media messages in Thai language with sentiment category (positive, neutral, negative, question)"""
+
+    _DOWNLOAD_URL = "https://github.com/PyThaiNLP/wisesight-sentiment/raw/master/huggingface/data.zip"
+    _TRAIN_FILE = "train.jsonl"
+    _VAL_FILE = "valid.jsonl"
+    _TEST_FILE = "test.jsonl"
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name="wisesight_thai_sentiment_source",
+            version=datasets.Version(_SOURCE_VERSION),
+            description="Wisesight Sentiment Corpus Source version (positive, neutral, negative, question)",
+            schema="source",
+            subset_id="wisesight_thai_sentiment",
+        ),
+        SEACrowdConfig(
+            name="wisesight_thai_sentiment_seacrowd_text",
+            version=datasets.Version(_SEACROWD_VERSION),
+            description="Wisesight Sentiment Corpus Seacrowd version (positive, neutral, negative, question)",
+            schema="seacrowd_text",
+            subset_id="wisesight_thai_sentiment",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "wisesight_thai_sentiment_source"
+
+    def _info(self):
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "texts": datasets.Value("string"),
+                    "category": datasets.features.ClassLabel(names=["pos", "neu", "neg", "q"]),
+                }
+            )
+        elif self.config.schema == "seacrowd_text":
+            features = schemas.text_features(["pos", "neu", "neg", "q"])
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage="https://github.com/PyThaiNLP/wisesight-sentiment",
+            citation=_CITATION,
```

Comment:
also add the license here
```suggestion
            license=_LICENSE,
            citation=_CITATION,
```

Reply:
Okay